src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -71,10 +68,13 @@ def set_scp_attachment(
     def clean_and_remove_policy_attachment(
             organization_mapping,
             path,
+            policy_filename,
             organizations,
             policy_type):
+        _type = 'scp' if policy_type == "SERVICE_CONTROL_POLICY" else 'tagging-policy'
```

Comment:
Would be good to change these hard coded strings into STATIC variables at the top.
Could you introduce this change?

Reply:
In the `apply` function, the variable `_type` uses the exact opposite, setting it to either `'SERVICE_CONTROL_POLICY' or 'TAG_POLICY'`. It would be easier to keep this consistent, and only tansition to the short version when we need it in an output.

So in other words, I would suggest to use the longer version all the time in the code. And only when we need to output it or set it as part of a path convert it to the print/path friendly version through calling a function that transforms it to that style.

If you change the next line into a function, you only need to pass the `policy_type` for this function.

Reply:
Could you change L100 to that style too? The one with `for policy in ['scp', 'tagging-policy']:`.
To something like:

```
for policy_type in [SERVICE_CONTROL_POLICY, TAGGING_POLICY]:
```

Reply:
I wasn't quite sure what you meant by "Would be good to change these hard coded strings into STATIC variables at the top.", but I made the rest of the suggested changes in the recent commit.